README.md

```diff
@@ -128,6 +128,7 @@ use {
             ["<cr>"] = "open",
             ["S"] = "open_split",
             ["s"] = "open_vsplit",
+            ["t"] = "open_tabnew",
```

Comment:
I try to keep the README mappings as a reflection of the default mappings. to that end, can you add this to the `defaults.lua` file as well?

It should also be listed in the help file in `docs/`
See `:h neo-tree-mappings`

Reply:
Done